### ISSUE TITLE
Update .ui-content-area area with correct svg

### ISSUE THREAD

**Issue by lmslachciak** [Samsung/TAU#778]
Update .ui-content-area area 
Curently .ui-content-area is using
 src/css/profile/mobile/theme-changeable/images/0_Round_corner/round.svg
it should use
bottom_left.svg
bottom_right.svg
top_left.svg
top_right.svg

**Comment by lslachciak** [comment]
file  src/css/profile/mobile/theme-changeable/images/0_Round_corner/round.svg shoul be deleted

**Comment by TomaszLukawskiSam** [comment]
This is problem with use mask-border-image with four images as sources.

**Comment by lslachciak** [comment]
yes, I see now, looks like it should be combined during build process..
